src/practices/Java/JavaGitignoreCorrectlySetPractice.ts

```diff
@@ -50,7 +51,7 @@ export class JavaGitignoreCorrectlySetPractice implements IPractice {
         return PracticeEvaluationResult.practicing;
       }
     }
-    return PracticeEvaluationResult.unknown;
+    throw ErrorFactory.newPracticeEvaluateError('Gitignore is partially correct');
```

Comment:
The error at the beginning of this function is correct, but here should be `notPracticing`?

Maybe the error should be thrown only in `else` block if no package manager has been found.

Reply:
Resolved in 4a5eb4be0a7bf8f4f98c53f6cff212ac9ca13529